testing/dbGenTests.test.js

```diff
@@ -43,8 +59,58 @@ for (let i = 0; i < 7; i += 1) {
     expect(Number(resTime) >= Number(open)).toBe(true);
     if (resTime.length > 2) {
       test(`but not between ${resTime[2]} and ${resTime[1]}`, () => {
-      expect(Number(resTime) <= Number(resTime[1]) && Number(resTime) >= Number(resTime[2])).toBe(false);
+        expect(Number(resTime) <= Number(resTime[1]) && Number(resTime) >= Number(resTime[2])).toBe(false);
       });
     }
   });
 }
+
+// test return date
+test('return date should return a date', () => {
+  const date = generator.returnDate(new Date('September 17, 2018'), 5);
+  expect(JSON.stringify(date)).toBe('"2018-09-22T07:00:00.000Z"');
+});
+
+// test formatDate
+test('format date should return a formatted string', () => {
+  expect(generator.formatDate(new Date('september 17, 2018'))).toBe('2018-8-17');
```

Comment:
How come you changed this just now from '2018-9-17'? Wasn't that correct?

Reply:
since I was concating numbers to string it was dropping the leading 0, so test was failing.
got rid of leading 0 as i don't think it is needed